airbyte-integrations/connectors/source-salesforce/unit_tests/api_test.py

```diff
@@ -52,7 +51,7 @@
 _NUMBER_OF_DOWNLOAD_TRIES = 5
 _FIRST_CALL_FROM_JOB_CREATION = 1
 
-_ANY_CATALOG = ConfiguredAirbyteCatalog.parse_obj({"streams": []})
+_ANY_CATALOG = ConfiguredAirbyteCatalogSerializer.load({"streams": []})
```

Comment:
I'll update that in the feature branch as I had to update that as well but since this was written, we've implemented a `CatalogBuilder` which abstract how we build a catalog so we could `CatalogBuilder().build()` here. 

Again, nothing to do in this PR as I'll fix it later so this comment is just for knowledge sharing purposes